### PR TITLE
Install nvim-qt executable + desktop file

### PR DESCRIPTION
**Pull request by nowrep**
Not really sure what to use as `Comment`, so I just copied it from Arch GVIM desktop file.

### CODE REVIEW

File: src/gui/nvim-qt.desktop [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Neovim
```
Comment (equalsraf) [inline src/gui/nvim-qt.desktop]: Probably better to go with Neovim-Qt here. Some people have wrappers for launching term+Neovim.

**Comment by nowrep** [comment]
Updated diff

**Comment by equalsraf** [comment]
Looks good, the only issue I'm seeing here is that make install is also installing libmsgpack shared libraries/headers/etc from the bundled msgpack. This should fix that

```diff
diff --git a/third-party/CMakeLists.txt b/third-party/CMakeLists.txt
index 329da96..2c086a4 100644
--- a/third-party/CMakeLists.txt
+++ b/third-party/CMakeLists.txt
@@ -23,7 +23,7 @@ endif()
 
 set(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR})
 set(MSGPACK_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/msgpack-c-cpp-1.0.0/)
-add_subdirectory(${MSGPACK_SOURCE_DIR})
+add_subdirectory(${MSGPACK_SOURCE_DIR} EXCLUDE_FROM_ALL)
 
 # Similar enough to FindMsgpack
 set(MSGPACK_INCLUDE_DIRS ${MSGPACK_SOURCE_DIR}/include PARENT_SCOPE)
```

**Comment by equalsraf** [comment]
Fixed the msgpack issue in master, this branch needs a rebase.

**Comment by nowrep** [comment]
Done

**Comment by equalsraf** [comment]
Merged into master, thanks @nowrep.

**Pull request merged**